src/bindings/python/src/pyopenvino/core/core.cpp

```diff
@@ -169,12 +169,13 @@ void regclass_Core(py::module m) {
     cls.def(
         "compile_model",
```

Comment:
Please update this function accordingly, here:
https://github.com/openvinotoolkit/openvino/blob/ed65d199bb3365c9bb296ce1ec1c1acda81b096a/src/bindings/python/src/openvino/runtime/ie_api.py#L357

and here:
https://github.com/openvinotoolkit/openvino/blob/ed65d199bb3365c9bb296ce1ec1c1acda81b096a/src/bindings/python/src/openvino/runtime/ie_api.py#L439